docs/sdk-json-proposal/SDK_API_EXAMPLES.md

```diff
@@ -0,0 +1,199 @@
+# Mesh Data Plane SDK – Examples
+
+## Video Sender Example
+
+```c
+int main(void)
+{
+    /* Default client configuration */
+    const char *client_config = R"(
+      {
+        "apiVersion": "v1",
```

Comment:
Can we split "ApiVersion" to major/minor key-value ?

Reply:
Let it sit in one string. You can define any version format in there like "v1.2.3.4" but it should be a single string.